liquibase-standard/src/main/java/liquibase/change/core/AddLookupTableChange.java

```diff
@@ -194,30 +194,54 @@ public SqlStatement[] generateStatements(Database database) {
         String existingTableCatalogName = getExistingTableCatalogName();
         String existingTableSchemaName = getExistingTableSchemaName();
 
-        SqlStatement[] createTablesSQL = {new RawSqlStatement("CREATE TABLE " + database.escapeTableName(newTableCatalogName, newTableSchemaName, getNewTableName()) + " AS SELECT DISTINCT " + database.escapeObjectName(getExistingColumnName(), Column.class) + " AS " + database.escapeObjectName(getNewColumnName(), Column.class) + " FROM " + database.escapeTableName(existingTableCatalogName, existingTableSchemaName, getExistingTableName()) + " WHERE " + database.escapeObjectName(getExistingColumnName(), Column.class) + " IS NOT NULL")};
+        StringBuilder createTableQuery = new StringBuilder(String.format("CREATE TABLE %s", database.escapeTableName(newTableCatalogName, newTableSchemaName, getNewTableName())))
+                .append(String.format(" AS SELECT DISTINCT %s", database.escapeObjectName(getExistingColumnName(), Column.class)))
```

Comment:
As you are using Stringbuilder, mixing it with String.format feels a bit strange. You could use just the String.format for everything as it improves readability. Right now looking at this query is complex and I need to parse many appends and formats to understand it.